### PR TITLE
Sumac

### PR DESCRIPTION
**Pull request by Henrrypg**
# Description

This intends to add support to Sumac Release.

### CODE REVIEW

**Review by Ian2012 — APPROVED** [review]

**Review by MoisesGSalas — COMMENTED** [review]
Remove the python3.8 classifiers.

Keep in mind this change: https://github.com/overhangio/tutor-mfe/pull/225

File: tutormfe_extensions/patches/openedx-cms-production-settings [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {%- if MFE_EXTENSIONS_BY_PATH %}
-{%- if get_mfe("course-authoring") %}
-COURSE_AUTHORING_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ CMS_HOST }}/course-authoring"
+{%- if get_mfe("authoring") %}
+COURSE_AUTHORING_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ CMS_HOST }}/authoring"
```
Comment (Ian2012) [inline tutormfe_extensions/patches/openedx-cms-production-settings:3]: Is it `authoring` or `course-authoring`

Comment (Henrrypg) [reply tutormfe_extensions/patches/openedx-cms-production-settings:3]: look at https://github.com/overhangio/tutor-mfe/pull/225

Comment (Ian2012) [reply tutormfe_extensions/patches/openedx-cms-production-settings:3]: I mean the final path `/authoring`

**Pull request merged**